CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 1.1.0
```

Comment:
```suggestion
## 2.0.0
```
Per https://semver.org/ I recommend bumping the major version, considering it's a breaking change that changes the pack config schema completely.

Reply:
Good call!

Reply:
Looks like Changelog version still needs a sync with the new pack.yaml version